File: ebanking-backend/src/main/java/net/njema/entities/Employee.java
package net.njema.entities;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@AllArgsConstructor @NoArgsConstructor @Data @Builder
public class Employee {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id ;
    String FirstName;
    String LastName;
    String cin;
    String gender;
    String email;
    int age;
    @ManyToOne
    Agency agency;

}

File: ebanking-frontend/src/app/model/employee.ts
import { Agency } from "./agency"

export interface Employee{ /*class with !*/
  "id": number
  "firstName": string
  "lastName": string
  "cin": string
  "gender": string
  "email":string
  "age": number
  "agency":Agency

}
export interface EmployeeDetails {
    id: string;
    currentPage: number;
    totalPage: number;
    sizePage: number;
    employees: Employee[];
  }